pypeit/core/telluric.py

```diff
@@ -0,0 +1,1223 @@
+
```

Comment:
That is a good question. I think they should probably be in PypeIt since the sensfunc for near-IR and red/optical setups depends on them. A lot of the front end is however not implemented yet in the scripts. I think the code to generate the telluric grids ought to be a separate repo, and we are working on that.